portfolio.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Alison's Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="stylesheets/styles.css">
+        <link rel="stylesheet" href="stylesheets/portfolio_styles.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous"><link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Hind:wght@300&family=Lato:ital,wght@0,300;0,900;1,300&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <a class="homelink" href="index.html"><i class="fas fa-home"></i></a>
+        <header> 
+            <h2>Body of work</h2>
+        </header>
+        <hr>
+        <nav class="port-container">
```

Comment:
any reason you went with a `nav` element here? My accessibility is rusty, but I'm wondering if this would confused a screen reader. I wonder if we should use a section instead maybe?